include/dlr_relayvm.h

```diff
@@ -45,6 +45,7 @@ class DLR_DLL RelayVMModel : public DLRModel {
   std::vector<std::vector<int64_t>> output_shapes_;
   DataTransform data_transform_;
   void InitModelPath(std::vector<std::string> paths);
+  void InitModelPath(ModelPath paths);
```

Comment:
This is probably not an appropriate constructor because not all attributes of `ModelPath` are applicable to RelayVM.
The factory method creating the various backends should be the one inspecting the struct and invoking the constructors of appropriate backends with the correct arguments.

Reply:
Thats ok, it shares `model_lib` and `metadata` fields in `ModelPath` with GraphRuntime. Plus we're just doing the same thing that the other `InitModelPath` does:
https://github.com/neo-ai/neo-ai-dlr/blob/5bb6d49c80a2733feb1187fb3a6077b9567f21b7/src/dlr_relayvm.cc#L14-L28